src/rgw/driver/rados/rgw_rest_user.cc

```diff
@@ -635,6 +677,10 @@ class RGWOp_Key_Create : public RGWRESTOp {
   RGWOp_Key_Create() {}
 
   int check_caps(const RGWUserCaps& caps) override {
+    int r = caps.check_cap("users-without-keys", RGW_CAP_WRITE);
```

Comment:
"users-without-keys" shouldn't grant access to RGWOp_Key_Create or RGWOp_Key_Remove